README.md

```diff
@@ -1,3 +1,33 @@
 # Go client and cli for [outline][def]
 
-[def]: https://www.getoutline.com/
\ No newline at end of file
+[def]: https://www.getoutline.com/
+
+
+
+# Usage
+
+
+## Client
```

Comment:
The goal of this project is to be an CLI.
Now we document the (http) client we use exclusively internally.
Does it make sense to document this somewhere else (or even not at all 😅 ) instead of the README? 

However, I really appreciate the documentation. It helped me a lot!

Reply:
Clarified in call.
All fine now ✔️